### PR TITLE
add unifi input

### PR DESCRIPTION
**Pull request by orevron**

### CODE REVIEW

File: src/inputs/unifi_input.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+                raise Exception('Authentication failed')
+        except Exception as e:
+            logging.error(f'Error authenticating: {e}')
+            raise e
+
+    def connect_to_websocket(self) -> None:
+        ws_url = self.host.replace('https', 'wss') + '/proxy/protect/ws/updates'
+        ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+        ssl_context.check_hostname = False
+        ssl_context.verify_mode = ssl.CERT_NONE
```
Comment (nivg1992) [inline src/inputs/unifi_input.py:46]: @orevron let's make that configurable

File: src/inputs/unifi_input.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    def configure(self, config):
+        self.host = config["host"]
+        self.user = config["user"]
+        self.password = config["password"]
+        self.authenticate()
+        self.get_bootstrap()
+
+
+    def get_streams(self):
+        return [{"name": camera["name"], "stream_url": self.get_rtsp_link(camera.get('channels').get('High').get('rtspAlias'))} for camera in self.cameras.values()]
```
Comment (nivg1992) [inline src/inputs/unifi_input.py:148]: also the image quality

**Review by nivg1992 — APPROVED** [review]
looks amazing, just few comments

File: src/inputs/unifi_input.py [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+            camera_name = self.get_camera_name(header)
+            logging.debug(f'[Unifi] Smart motion start detected for name: {camera_name}')
+            super().start_capture_topic(camera_name)
+        elif payload and is_smart_detection == False:
+            camera_name = self.get_camera_name(header)
+            logging.debug(f'[Unifi] Smart motion stop detected for name: {camera_name}')
+            super().stop_capture_topic(camera_name)
+
+    def get_camera_name(self, header) -> None:
+        return f"{self.cameras[header.get('id')].get('name')}_{PF_UNIFI_IMAGE_QUALITY}".lower()
```
Comment (nivg1992) [inline src/inputs/unifi_input.py:143]: There is already a configuration file, I think that it could be better to take it from there

Comment (orevron) [reply src/inputs/unifi_input.py:143]: what config file you're talking about? cameras.json?

Comment (nivg1992) [reply src/inputs/unifi_input.py:143]: sure, the cameras.json

I think for the first phase we could configure the same image quality for all the cameras

**Pull request merged**